### PR TITLE
Add the `osu!monthly, August 2022` newspost

### PR DESCRIPTION
**Pull request by 0x84f**
ya

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] ~~*(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)~~

### CODE REVIEW

**Comment by Ephemeralis** [comment]
I've applied a fairly fat set of changes that gets things into the correct format (especially pretty much all of the YouTube videos), adds in some missing content, expands on a few sections and makes things more readable while preserving the original prose (mostly).

Feel free to make changes as needed!

**Comment by 0x84f** [comment]
The quiz has been privately reviewed by Ephemeral and Walavouchey, they will be added before merge

File: news/2022-09-01-osumonthly-2.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+- [Things to look out for in September](#things-to-look-out-for-in-september)
+- [End word](#end-word)
+
+## What's new in osu!(lazer)
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/RxRwqXmMxd4?list=PLmWVQsxi34bMTmgpDArfxJ32Q0mBLhcE-" frameborder="0" allowfullscreen></iframe>
+
+The weekly updates for osu!(lazer) continue, most notably featuring a change to the osu! slider animations, which should now more closely match the stable version of the game and hopefully minimise the "weird feeling" players have reported when playing the latest version.
+
+Furthermore, a bunch of new features and fixes have been added into the editor, which will make mappers' lives easier as [OliBomby](https://osu.ppy.sh/users/6573093) continues to implement the features from his ever-popular [Mapping Tools](https://osu.ppy.sh/community/forums/topics/940368?n=1) utility directly into osu!(lazer) proper.
```
Comment (Walavouchey) [inline news/2022-09-01-osumonthly-2.md:32]: i highly recommend mentioning the slider splitting (coming next release) and merging features, and including a video of them

Comment (0x84f) [reply news/2022-09-01-osumonthly-2.md:32]: Upon our conversation ([from here](https://discord.com/channels/188630481301012481/218677502141399041/1013863621492752494)) and Blushing's thoughts I decided it's better to leave it for next month (unless by some slim chance it releases before the 1st) and have a portion showing it off within the lazer section of the article.

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**